eng/Build-Native.cmd

```diff
@@ -172,11 +172,6 @@ if not exist "%__DotNetCli%" (
     echo %__MsgPrefix%Assertion failed: dotnet cli not found at path "%__DotNetCli%"
     exit /b 1
 )
-set __MSBuildPath=%__ProjectDir%\.dotnet\sdk\2.1.505\MSBuild.dll
-if not exist "%__MSBuildPath%" (
-    echo %__MsgPrefix%Assertion failed: dotnet cli sdk not found at path "%__MSBuildPath%"
-    exit /b 1
-)
 
 set __DotNetRuntimeVersion=2.1.9
```

Comment:
Since you are upgrading the SDK to 3.0.x, this doesn't match what it meant to be.  The SOS tests assume that this variable is set to the SDK runtime version and assumes it is a 2.1.x runtime.

The tests are probably not testing against a 2.1.x runtime but 3.0.x twice.